### ISSUE TITLE
accuracy function does not calculate MASE

### ISSUE THREAD

**Issue by sbcalaff** [robjhyndman/forecast#749]
"accuracy" function does not calculate "MASE" measure if input "f" is a numerical vector containing forecasts. "accuracy" function does calculate "MASE" only if input "f" is a "forecast" object.

Already seen my mistake:
> By default, the MASE calculation is scaled using MAE of training set naive
> forecasts for non-seasonal time series, training set seasonal naive forecasts
> for seasonal time series and training set mean forecasts for non-time series data.
> If \code{f} is a numerical vector rather than a \code{forecast} object, the MASE
> will not be returned as the training data will not be available.

**Comment by mitchelloharawild** [comment]
Glad that it is documented :smile:
